File: cpp_results/instruct_generations/cpp/files_after_feedback/question-4-pass-1.cpp
#include <bits/stdc++.h>
using namespace std;
int check(int A, int B) {
    return (B * 10 + B % 10) % 10 == (A % 10);
}
int main() {
    int A, B;
    cin >> A >> B;
    if (check(A, B)) {
        cout << "Yes" << endl;
    } else {
        cout << "No" << endl;
    }
    return 0;
}

File: results/fine_tuning/test_orig_files/question-4-pass-1.py
import math



def check(A, B):
    if A == B:
        return True
    if A < B:
        B = B * 10
        while B!= A:
            B = B // 10
            B = B * 10 + A % 10
            A = A // 10
    else:
        A = A * 10
        while A!= B:
            A = A // 10
            A = A * 10 + B % 10
            B = B // 10
    return A == B
